UnitTests/TestSuites/ImagesTests.swift

```diff
@@ -80,5 +80,27 @@ class ImagesTests: XCTestCase {
 
     let expected = self.fixtureString("Images-File-CustomName.swift.out")
     XCTDiffStrings(result, expected)
-  }
+	}
+
+	func testFileWithDotSyntax() {
```

Comment:
I don't know, sometimes Xcode forgets them, or something else.

Personally I always use tabs so that's why it sometimes gets through, even though I try to be careful.